package.json

```diff
@@ -38,28 +38,28 @@
         "@rollup/plugin-typescript": "^11.0.0",
         "@types/babel__core": "^7.20.0",
         "@types/glob": "^8.1.0",
-        "@types/jest": "^29.4.0",
-        "@types/node": "^18.14.6",
-        "@typescript-eslint/eslint-plugin": "^5.54.1",
-        "@typescript-eslint/parser": "^5.54.1",
+        "@types/jest": "^29.5.0",
+        "@types/node": "^18.15.5",
+        "@typescript-eslint/eslint-plugin": "^5.56.0",
+        "@typescript-eslint/parser": "^5.56.0",
         "bytes": "^3.1.2",
         "es5-proxy-compat": "^0.22.4",
-        "eslint": "^8.35.0",
+        "eslint": "^8.36.0",
         "eslint-plugin-import": "^2.27.5",
         "eslint-plugin-jest": "^27.1.7",
-        "fs-extra": "^11.1.0",
-        "glob": "^8.1.0",
+        "fs-extra": "^11.1.1",
+        "glob": "^9.3.1",
```

Comment:
Based on the [release notes for glob 9.0.0](https://github.com/isaacs/node-glob/blob/main/changelog.md#90), as well as the fact that we are using `glob.sync` in [several of our scripts](https://github.com/search?q=repo%3Asalesforce%2Flwc%20glob.sync&type=code) (e.g. releasing a new version, running benchmarks), and given that [`glob.sync` does not seem to be exported anymore](https://unpkg.com/browse/glob@9.3.1/dist/cjs/glob.d.ts), this 9.0 upgrade seems likely to break in a way that would not be detected by our CI tests.